sklearn/tests/test_naive_bayes.py

```diff
@@ -557,7 +557,7 @@ def test_cnb():
     Y = np.array([0, 0, 0, 1])
 
     # Verify inputs are nonnegative.
-    clf = ComplementNB(alpha=1.0)
+    clf = ComplementNB(alpha=1.0, norm=True)
```

Comment:
Now `ComplementNB` is only tested with `norm=True`.
We would need a test with `norm=False`

Reply:
@TomDLT - added.